### DISCUSSION TITLE
Failed to read existing file Error

### DISCUSSION THREAD

**Discussion by HollaDieWaldfee2022** [jomjol/AI-on-the-edge-device#1449] [Q&A] [answered]
Hi,

I am already using on eESP32 with AI on the edge for my watermeter. Now I wanted to install one for the gasmeter.
I set everything up and now I get the error:
Failed to read existing file

in some parts of the menu.

![grafik](https://user-images.githubusercontent.com/109627421/205075536-3a431e97-98e9-406f-9511-35a2c0e98406.png)

There are also some strange thing in the config menu:
![grafik](https://user-images.githubusercontent.com/109627421/205075779-49e98924-eb16-4219-b091-df4cc46e9c21.png)

Also the info menu isnt working right:
![grafik](https://user-images.githubusercontent.com/109627421/205076065-207c6785-4e73-43ed-b005-39c6603ceb61.png)

I am using a 8GB intenso card (fat32 on windows).

Please can anybody help me?

Best regard and thank you
Ben

**Comment by caco3** [comment]
which version are you using?

Try to reinstall it resp. use the latest version.

Maybe your SD-Card is not setup properly?

**Comment by HollaDieWaldfee2022** [comment, marked as answer]
I seems to be solved with the current version 13.0.4.

Thanks a lot
